server/container/guice/cassandra-guice/src/test/resources/mailqueueview.properties

```diff
@@ -0,0 +1,3 @@
+bucketCount=1
```

Comment:
from the outside "mailqueueview" doesn't make sense. mailqueue is probably enough